### PR TITLE
File Added : All changed files

### PR DESCRIPTION
**Pull request by Oshin-250197**

### CODE REVIEW

**Review by roshnidurugwar — APPROVED** [review]

**Review by arshad05786 — APPROVED** [review]

**Review by SamikshaWani — APPROVED** [review]

**Review by roshnidurugwar — APPROVED** [review]

**Review by yogesh141293 — APPROVED** [review]

File: Oshin/html/Assignment1/Question1.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+<body>
+<h1>
+    HTML
+</h1>
+<h2>
+    Introduction
+</h2>
+<p>HTML is a standard markup language for web pages.</p>
+</body>
+</html>
```
Comment (Pawan1005) [inline Oshin/html/Assignment1/Question1.html:16]: need indentation
add new line at the end of file

File: Oshin1/html/Assignment1/Question1.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+<body>
+<h1>
+    HTML
+</h1>
+<h2>
+    Introduction
+</h2>
+<p>HTML is a standard markup language for web pages.</p>
+</body>
+</html>
```
Comment (Pawan1005) [inline Oshin1/html/Assignment1/Question1.html]: need indentation
add new line at the end of file

File: Oshin1/html/Assignment1/Question2.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+</h2>
+<h3>
+    Notepad or TextEdit
+</h3>
+    <p>
+        Web Pages can be created and modified by using professional HTML editors.<br>
+        However,for learning HTML we recommend a simple text editor like Notepad(PC) or TextEdit(Mac).<br>
+    </p>
+</body>
+</html>
```
Comment (Pawan1005) [inline Oshin1/html/Assignment1/Question2.html]: need indentation  
add new line at the end of file

File: Oshin1/html/Assignment1/Question3.html [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+<h3>
+    Notepad or TextEdit
+</h3>
+    <p>
+        Web Pages can be created and modified by using professional HTML editors.<br>
+        However,for learning HTML we recommend a simple text editor like Notepad(PC) or TextEdit(Mac).<br>
+    </p>
+</hr>
+</body>
+</html>
```
Comment (Pawan1005) [inline Oshin1/html/Assignment1/Question3.html]: need indentation
add new line at the end of file

File: Oshin1/html/Assignment1/Question4.html [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    <u>Educational Qualification </u>
+</h2>
+<p>
+    I have completed graduation in B.E Information Technology.<br>
+    I am 2019 passout .<br>
+    Currently i am looking for a job <br>
+</p>
+</body>
+</head>
+</html>
```
Comment (Pawan1005) [inline Oshin1/html/Assignment1/Question4.html]: need indentation
add new line at the end of file

File: Oshin1/html/Assignment2/Assign2.html [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+ <p>By some accounts,they disagreed about nearly everthing during that first meeting,but by the following year they struck a <SPAN STYLE="background-color:yellow">partnership.</SPAN> <br>
+Working from their dorm rooms,they build a search engine that used links to determine the importance of individual pages on the  <SPAN STYLE="color:blue">World Wide Web.</SPAN><br>
+<p>They called this search engine <SPAN STYLE="background-color:blue"><i>Backrub.</i></SPAN></p>
+<hr>
+<h2>Initial public offering</h2>
+<p>Google's initial public offering(IPO) took place five years later,on 19<sup>th</sup> August 2004.<br>
+At that time Larry Page,Sergey Brin,and Eric Schmidt agreed to work together at Google for <del>30</del><u>20</u>years, until 2024</p>
+<h1 style="text-align: center;color:green;">Thank You!!</h1>
+</body>
+</html>
```
Comment (Pawan1005) [inline Oshin1/html/Assignment2/Assign2.html]: need indentation
add new line at the end of file

File: Oshin1/html/Assignment2/Assign2.html [hunk 7]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<head>
```
Comment (Pawan1005) [inline Oshin1/html/Assignment2/Assign2.html]: opening head tag is missing

File: Oshin1/html/Assignment3/Question1.html [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+                    <ul style="list-style-type: disc;">
+                        <li>Payal</li>
+                        <li>Sakshi</li>
+                    </ul>
+            </ol>
+        </li>
+        </ol>
+    </div>
+    </body>
+</html>
```
Comment (Pawan1005) [inline Oshin1/html/Assignment3/Question1.html]: need indentation
add new line at the end of file

File: Oshin1/html/Assignment3/Secondquestion/MI-MEANpage.html [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+                color:indigo
+            }
+            a:visited
+            {
+                color: indigo;
+            }
+        </style>
+    </head>
+    <h1 style="text-align: center;">Ambersoft</h1>
+
```
Comment (Pawan1005) [inline Oshin1/html/Assignment3/Secondquestion/MI-MEANpage.html]: remove extra line

File: Oshin1/html/Assignment3/Secondquestion/MI-MEANpage.html [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+        </div>
+        <div>
+            <ol style="list-style-type: lower-roman;">
+                <li> Naresh </li>
+                <li>Urmila</li>
+                <li>Pari</li>
+            </ol>
+        </div>
+    </body>
+</html>
```
Comment (Pawan1005) [inline Oshin1/html/Assignment3/Secondquestion/MI-MEANpage.html]: add new line at the end of file

File: Oshin1/html/Assignment3/Secondquestion/contactpage.html [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+            </div>
+            <div id="texts" style="float:left;"> 
+                <h2 style="color: palevioletred;" text align="center">Contact Details</h2><br>
+                <hr>
+                Contact number : 1234567890<br>
+                Website: http://www.ambersoft.in<br>
+                Address: G-704 Cappriccio Apt, Wakad Road, Wakad Pune MH 411057 IN<br>
+            </div>
+        </div>
+    </body>
```
Comment (Pawan1005) [inline Oshin1/html/Assignment3/Secondquestion/contactpage.html:41]: closing HTML tag is missing 
add new line at the end of file

File: Oshin1/html/Assignment3/Secondquestion/homepage.html [hunk 12]
```diff
@@ -0,0 +1,55 @@
...
+        AmberSoft was earlier known as KnowledgeQuest Infotech.
+        In that life we have been doing recruitment, software development as well as product development.
+        Our software business was acquired by a US company called sTec in March 2011.
+        Our CEO Anurag Agarwal, was heading India operations for sTec.
+        In May, 2013, he decided to move out of sTec and restart the company. 
+        And in this life we have taken name AmberSoft.
+    </p>
+    </div>
+  </body>
+  </html>
```
Comment (Pawan1005) [inline Oshin1/html/Assignment3/Secondquestion/homepage.html:55]: need indentation
add new line at the end of file

File: Oshin1/html/Assignment4/Question1.html [hunk 13]
```diff
@@ -0,0 +1,50 @@
...
+            </tr>
+            <tr>
+                <th>Females</th>
+                <td>1.7</td>
+                <td>0.002</td>
+                <td>43%</td>
+            </tr>  
+    </table>    
+</body>
+</html>
```
Comment (Pawan1005) [inline Oshin1/html/Assignment4/Question1.html]: need indentation
add new line at the end of file

File: Oshin1/html/Assignment4/Question2.html [hunk 14]
```diff
@@ -0,0 +1,91 @@
...
+        </tr>
+        <tr>
+            <th class="c">4.30</th>
+            <td class="c"> </td>
+            <td class="c"> </td>
+            <td class="c"> </td>
+        </tr>
+    </table>  
+</body>
+</html>
```
Comment (Pawan1005) [inline Oshin1/html/Assignment4/Question2.html]: need indentation
add new line at the end of file

File: Oshin1/html/Assignment4/Question3.html [hunk 15]
```diff
@@ -0,0 +1,62 @@
...
+            </tr>
+            <tr>
+                <td>Wednesday</td>
+                <td style="background-color:bisque">8:00 a.m.</td>
+                <td style="background-color: aqua;">12:00 p.m.</td>
+                <td>XSL Formatting Objects</td>
+            </tr>
+        </table>
+    </body>
+</html>
```
Comment (Pawan1005) [inline Oshin1/html/Assignment4/Question3.html]: add new line at the end of file

**Review by Pawan1005 — CHANGES_REQUESTED** [review]
need indentation
add new line at the end of file

File: Oshin1/html/Assignment3/Secondquestion/Homepage.html [hunk 16]
```diff
@@ -0,0 +1,49 @@
...
+        We help our clients achieve greater organizational performance by assessing, recruiting and developing the best and brightest people for their business.
+        AmberSoft was earlier known as KnowledgeQuest Infotech.
+        In that life we have been doing recruitment, software development as well as product development.
+        Our software business was acquired by a US company called sTec in March 2011.
+        Our CEO Anurag Agarwal, was heading India operations for sTec.
+        In May, 2013, he decided to move out of sTec and restart the company. 
+        And in this life we have taken name AmberSoft.
+        </p>
+        </body>
+</html>
```
Comment (Pawan1005) [inline Oshin1/html/Assignment3/Secondquestion/Homepage.html:44]: indentation

**Review by Pawan1005 — APPROVED** [review]

**Review by SnehalAhir — APPROVED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**